benchmark_stream.go

```diff
@@ -85,11 +92,34 @@ func (b *streamIOBenchmark) NextRequest() ([]byte, error) {
 // HandleResponse records stream response.
 func (b *streamIOBenchmark) HandleResponse(res []byte) error {
 	b.streamResponses = append(b.streamResponses, res)
+
+	b.streamCallResult.streamMsgsReceived++
+
 	return nil
 }
 
-func newStreamIOBenchmark(streamRequests [][]byte) *streamIOBenchmark {
+func newStreamIOBenchmark(streamRequests [][]byte, r *streamCallResult) *streamIOBenchmark {
 	return &streamIOBenchmark{
-		streamRequests: streamRequests,
+		streamRequests:   streamRequests,
+		streamCallResult: r,
 	}
 }
+
+type streamCallResult struct {
+	latency time.Duration
+
+	streamMsgsSent     int
+	streamMsgsReceived int
+}
```

Comment:
Please be consistent, everywhere else in the code we use the entire word `messages` and not `msgs` ;)

Reply:
My bad :-)